app/charts/map/ref.ts

```diff
@@ -0,0 +1,13 @@
+import { Map } from "mapbox-gl";
```

Comment:
I think it's helpful here to add a small comment to show that we are not so happy with the solution.

```
/**
 * We need to access the map from the map controls. Until we have a good solution for
 * sibling components, we use a global non observable ref.
 */
```